File: src/test/java/com/wxl/crawlerdytt/core/impl/HttpClientDownLoaderTest.java
package com.wxl.crawlerdytt.core.impl;

import com.wxl.crawlerdytt.core.DyttConstants;
import org.apache.http.impl.client.CloseableHttpClient;
import org.apache.http.impl.client.HttpClients;
import org.junit.Test;

import java.io.IOException;

/**
 * Create by wuxingle on 2020/5/2
 */
public class HttpClientDownLoaderTest {


    @Test
    public void test1() throws IOException {
        CloseableHttpClient httpClient = HttpClients.createDefault();
        HttpClientDownLoader downLoader = new HttpClientDownLoader(httpClient);

        String download = downLoader.download("https://www.dytt8.net/", DyttConstants.DEFAULT_CHARSET);

        System.out.println(download);
    }
}

File: src/test/kotlin/com/wxl/dyttcrawler/downloader/HttpClientDownloaderTest.kt
package com.wxl.dyttcrawler.downloader

import com.wxl.dyttcrawler.downloadPage
import com.wxl.dyttcrawler.getDownloader
import org.junit.jupiter.api.Test
import us.codecraft.webmagic.Request
import us.codecraft.webmagic.Site
import us.codecraft.webmagic.Task

/**
 * Create by wuxingle on 2021/10/02
 * -Djavax.net.debug=ssl,handshake
 */
class HttpClientDownloaderTest {

    @Test
    fun test1() {
        val html = downloadPage("http://www.dytt8.net").html
        println(html)
    }

}